cache/cache_test.cc

```diff
@@ -25,16 +25,31 @@
 namespace ROCKSDB_NAMESPACE {
 
 // Conversions between numeric keys/values and the types expected by Cache.
-static std::string EncodeKey(int k) {
+static std::string EncodeKey16Bytes(int k) {
   std::string result;
   PutFixed32(&result, k);
+  result.append(std::string(12, 'a'));  // Add a 12B padding.
   return result;
 }
-static int DecodeKey(const Slice& k) {
+
+static int DecodeKey16Bytes(const Slice& k) {
+  assert(k.size() == 16);
+  return DecodeFixed32(std::string(k.data(), 4).c_str());
```

Comment:
Unnecessary temporaries and misleading c_str. `DecodeFixed32(k.data())` should be fine. Recommend comment that padding is ignored.

Reply:
Will do.